api/indexer.oas2.json

```diff
@@ -1604,6 +1702,43 @@
         }
       }
     },
+    "Box": {
+      "description": "Box name and its content.",
+      "required": [
+        "name", 
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "\\[name\\] box name, base64 encoded",
+          "format": "byte",
+          "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$",
```

Comment:
The pattern can be left out. I think it became deprecated when we stopped using gorilla mux.

Reply:
removed and resolving